test/integration/integration_test.go

```diff
@@ -279,31 +288,57 @@ func checkConfigKey(configPath string, condition string, keyPath string) error {
 
 func compareExpectedWithActualContains(expected string, actual string) error {
 	if !strings.Contains(actual, expected) {
-		return fmt.Errorf("Output did not match. Expected: %s, Actual: %s", expected, actual)
+		return fmt.Errorf("Output did not match. Expected: '%s', Actual: '%s'", expected, actual)
```

Comment:
Personally I find _errors.New_ more concise than _fmt.Errorf_, but the end result is probably the same.

+1 for the quotes around '%s' though. I had more than one occasion where having quotes helps to distinguish cases where I was not sure whether something was not working in general or whether the actual string was empty.